.github/workflows/ci.yml

```diff
@@ -173,6 +173,19 @@ jobs:
           name: nuget-package
           path: ./Bin/Release/ILGPU.*.*nupkg
 
+  publish-nuget-pre-release:
+    if: github.ref == 'refs/heads/master'
+    needs: [check-style, build-and-test]
+    runs-on: windows-latest
+    steps:
+      - name: Download NuGet package artifact
+        uses: actions/download-artifact@v2
+        with:
+          name: nuget-package
```

Comment:
same as the previous PR, `nuget-package` artifact doesn't exist on `m4rs-mt:master` 👀

Reply:
updated this one, nice catch, ty ⚡